README.md

```diff
@@ -98,6 +108,45 @@ Use the following settings to update cell execution time for a notification and
 
 ![notification](https://user-images.githubusercontent.com/3497137/111881088-01db5200-897d-11eb-8faa-4701cabfcde4.gif)
 
+## (Option) Notifications using `ntfy`
+
+You can recieve notifications via `ntfy`.
+
+**ntfy 2.7.0 documentation** https://ntfy.readthedocs.io/en/latest/
+
+> ntfy brings notification to your shell. It can automatically provide desktop notifications when long running commands finish or it can send push notifications to your phone when a specific command finishes.
```

Comment:
```suggestion
> ntfy brings notification to your shell. It can automatically provide desktop notifications when long running code executions finish or it can send push notifications to your phone when a specific execution finishes.
```